OOP/assignment5/Entity.java

```diff
@@ -0,0 +1,73 @@
+package socialNetworking;
+
+import java.util.ArrayList;
+
+public class Entity extends Node implements Record {
+
+	private String email;
+	private String describtion;
+	private String phoneNo;
+	private ArrayList<Entity> friends = new ArrayList<Entity>();
```

Comment:
ACTION: Why are u still using List of Entity here? I already mentioned that 'friends' is not a part of node. What is the use of connection then?